docs/docs/developers/guides/js_apps/pay_fees.md

```diff
@@ -81,6 +81,41 @@ The claim object:
 
 #include_code claim_type_amount yarn-project/aztec.js/src/api/ethereum/portal_manager.ts javascript
 
+#### Half-bridging fee-juice in the Sandbox
+
+Bridging of the fee-juice asset is described fully [here](../../../developers/tutorials/codealong/contract_tutorials/token_bridge#deposit-to-aztec), but since we're going to claim and pay in one, we only need to mint/approve on L1.
+
+1. Get node info and create a public client pointing to the sandbox's foundry L1:
+
+#include_code get_node_info_pub_client yarn-project/end-to-end/src/spartan/smoke.test.ts javascript
+
+Notes: Steps 1 and 3 make use of viem functions, so be sure to: `import { createPublicClient, createWalletClient } from 'viem';`
+
+2. Now use these to create the L1FeeJuicePortalManager and get its L1TokenManager
+
+```
+const feeJuicePortalManager = new L1FeeJuicePortalManager( // Create an L1FeeJuicePortalManager
+    info.l1ContractAddresses.feeJuicePortalAddress,
+    info.l1ContractAddresses.feeJuiceAddress,
+    publicClient, // from step 1
+    getL1WalletClient(foundry.rpcUrls.default.http[0], 0),
+    createLogger('example:bridging-fee-juice');,
+);
+const tokenManager = feeJuicePortalManager.getTokenManager();
+```
+
+3. Now mint and approve the amount of tokens on L1
```

Comment:
You should be able to get rid of this section (point 3) and replace with [this](https://github.com/AztecProtocol/aztec-packages/blob/c7dc5492c431ad6052a92d7de265f2a2e59af728/yarn-project/aztec.js/src/api/ethereum/portal_manager.ts#L136-L137) 1 function call.